Resources/ProxyProvider/DIRACCAProxyProvider.py

```diff
@@ -1,18 +1,52 @@
 """ ProxyProvider implementation for the proxy generation using local (DIRAC)
     CA credentials
+
+    This class is a simple, limited CA, its main purpose is to generate a simple proxy for DIRAC users
+    who do not have any certificate register on the fly.
+
+    Required parameters in the DIRAC configuration for its implementation::
+
+    <Provider Name> section::
```

Comment:
Double colons create a "verbatim environment", but the next section is a bullet list. So only a single colon, and the bullet list indented to the same level is the preceding text (removing two spaces from each line with "*")